Constitution.txt

```diff
@@ -1,490 +1,264 @@
-
-18: To make all Laws which shall be necessary and proper for carrying into Execution the foregoing Powers, and all other Powers vested by this Constitution in the Government of the United States, or in any Department or Officer thereof.
+13: To make all Laws which shall be necessary and proper for carrying into Execution the foregoing Powers, and which shall be within the scope of the foregoing Powers. It shall be the duty of all Representatives and Senators, the President of these United States, all State Executives, and State Legislatures, to ensure before their approval that a Bill under consideration falls within the scope of the foregoing powers, and is not in violation of any term of this Constitution.
 
 Section 9
-1: The Migration or Importation of such Persons as any of the States now existing shall think proper to admit, shall not be prohibited by the Congress prior to the Year one thousand eight hundred and eight, but a Tax or duty may be imposed on such Importation, not exceeding ten dollars for each Person.
+1: The Privilege of the Writ of Habeas Corpus shall not be suspended, unless when in Cases of Rebellion or Invasion the public Safety may require it, whereupon it shall be suspended for more than one week only with the approval of three-fourths of both the House of Representatives and the Senate.
 
-2: The Privilege of the Writ of Habeas Corpus shall not be suspended, unless when in Cases of Rebellion or Invasion the public Safety may require it.
+2: No Bill of Attainder or ex post facto Law shall be passed.
 
-3: No Bill of Attainder or ex post facto Law shall be passed.
+3: No Capitation, or other direct, Tax shall be laid.
 
-4: No Capitation, or other direct, Tax shall be laid, unless in Proportion to the Census or Enumeration herein before directed to be taken.7
+4: No Tax or Duty shall be laid on Articles exported from any State, unless such Tax or Duty shall be imposed uniformly upon all States.
 
-5: No Tax or Duty shall be laid on Articles exported from any State.
+5: No Preference shall be given by any Regulation of Commerce or Revenue to the Ports of one State over those of another: nor shall Vessels bound to, or from, one State, be obliged to enter, clear, or pay Duties in another.
 
-6: No Preference shall be given by any Regulation of Commerce or Revenue to the Ports of one State over those of another: nor shall Vessels bound to, or from, one State, be obliged to enter, clear, or pay Duties in another.
+6: No Money shall be drawn from the Treasury, but in Consequence of Appropriations made by Law; and a regular Statement and Account of the Receipts and Expenditures of all public Money shall be published at least once per year. Such Statement shall include a record of any debts or surpluses arising from any individual Bill.
```

Comment:
The people should know which parts of the government are saving or losing money. By giving a specific time period, I make the government constitutionally more accountable, and by requiring accounting on a per-bill level, I give the people and their Representatives more ability to properly review the cost-effectiveness of a given law.